### PR TITLE
Release AC 1.10.7-18 and 1.10.10-8

### PR DESCRIPTION
**Pull request by kaxil**
Astronomer Certified 1.10.10-8, 2021-04-27
--------------------------------------------

Tag: https://github.com/astronomer/airflow/releases/tag/v1.10.10%2Bastro.8

### Bug Fixes

- Fix `sync-perm` to work correctly when update_fab_perms = False (#14847) ([commit](https://github.com/astronomer/airflow/commit/7b45e4d36))
- Webserver: Sanitize string passed to origin param (#14738) ([commit](https://github.com/astronomer/airflow/commit/825f250a0))
- Correctly parse (and copy) extras with python version to metadist ([commit](https://github.com/astronomer/airflow/commit/9dbcf77a9))

Astronomer Certified 1.10.7-18, 2021-04-27
--------------------------------------------

Tag: https://github.com/astronomer/airflow/releases/tag/v1.10.7%2Bastro.18

### Bug Fixes

- Fix `sync-perm` to work correctly when `update_fab_perms = False` (#14847) ([commit](https://github.com/astronomer/airflow/commit/8e49e8f22))
- Webserver: Sanitize string passed to origin param (#14738) ([commit](https://github.com/astronomer/airflow/commit/c0c4ff8b3))
- Correctly parse (and copy) extras with python version to metadist ([commit](https://github.com/astronomer/airflow/commit/38de13b67))

### CODE REVIEW

File: 1.10.10/alpine3.10/build-time-pip-constraints.txt [hunk 1]
```diff
@@ -0,0 +1,343 @@
+adal==1.2.5
```
Comment (kaxil) [inline 1.10.10/alpine3.10/build-time-pip-constraints.txt:1]: The constraints are from the `1.10.10-7-alpine3.10` image

File: 1.10.10/buster/build-time-pip-constraints.txt [hunk 2]
```diff
@@ -0,0 +1,186 @@
+alembic==1.4.3
```
Comment (kaxil) [inline 1.10.10/buster/build-time-pip-constraints.txt:1]: The constraints are from the `1.10.10-7-buster` image

File: 1.10.7/alpine3.10/build-time-pip-constraints.txt [hunk 3]
```diff
@@ -0,0 +1,334 @@
+adal==1.2.5
```
Comment (kaxil) [inline 1.10.7/alpine3.10/build-time-pip-constraints.txt:1]: The constraints are from the `1.10.7-7-alpine3.10` image

File: 1.10.7/buster/build-time-pip-constraints.txt [hunk 4]
```diff
@@ -0,0 +1,186 @@
+alembic==1.4.3
```
Comment (kaxil) [inline 1.10.7/buster/build-time-pip-constraints.txt:1]: The constraints are from the `1.10.10-7-buster` image

Comment (kaxil) [reply 1.10.7/buster/build-time-pip-constraints.txt:1]:
```
docker run -it quay.io/astronomer/ap-airflow:1.10.10-7-alpine3.10 /bin/bash -c "pip freeze"
```

File: 1.10.10/buster/build-time-pip-constraints.txt (hunk 2 above)
Comment (kaxil) [reply 1.10.10/buster/build-time-pip-constraints.txt:1]:
```
docker run -it quay.io/astronomer/ap-airflow:1.10.10-7-buster /bin/bash -c "pip freeze"
```

File: 1.10.10/alpine3.10/build-time-pip-constraints.txt (hunk 1 above)
Comment (kaxil) [reply 1.10.10/alpine3.10/build-time-pip-constraints.txt:1]:
```
docker run -it quay.io/astronomer/ap-airflow:1.10.10-7-alpine3.10 /bin/bash -c "pip freeze"
```

File: 1.10.7/alpine3.10/build-time-pip-constraints.txt (hunk 3 above)
Comment (kaxil) [reply 1.10.7/alpine3.10/build-time-pip-constraints.txt:1]:
```
docker run -it quay.io/astronomer/ap-airflow:1.10.7-17-alpine3.10 /bin/bash -c "pip freeze"
```

**Review by ashb — APPROVED** [review]

**Review by jedcunningham — APPROVED** [review]

**Pull request merged**